packages/dashboard/src/app/views/constants.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const TEMPLATE_META_DATA_TYPES = {
+  VERTICAL: 'VERTICAL',
+  COLOR: 'COLOR',
+  TAG: 'TAG',
```

Comment:
💭 If we're going to be adding custom taxonimes, then I don't think this should remain static 🤔 

But part of my confusion here is my lack of understanding about how often custom taxonomies will be added. If it's pretty static then that's gucci, but we'll need to make sure that if a taxonomy is ever added that we know to come update this stuff too

Reply:
This is not related to taxonomies..?

This refers to the `tag` key in a template's `raw.json` file.

Reply:
oh I see 😮‍💨 thanks for the clear up!